FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Core.UnitTests/DataProcessor/DataConverter/DicomDefaultSchemaConverterTests.cs

```diff
@@ -110,7 +111,7 @@ public void GivenAValidBasicSchema_WhenConvert_CorrectResultShouldBeReturned()
         {
             var result = _testDefaultConverter.Convert(
                 CreateTestJsonBatchData(_testMetadata),
-                "dicom");
+                DicomConstants.DicomResourceType);
```

Comment:
I think we should use hard code `Dicom` instead of `DicomConstants.DicomResourceType` in tests to make sure the name isn't modified by mistake.

Reply:
Here it is not used as expected in assert, just as a parameter. In expected I think we should use hard-coded one.

Reply:
I can update it to the hard-coded one as well.

Reply:
Updated to the hard-coded one.